### ISSUE TITLE
waypoing_recorder store to datacentre

### ISSUE THREAD

**Issue by cburbridge** [strands-project/autonomous_patrolling#60]
Currently the waypoint_recorder package stores points into a csv file. This file then has to be entered into the DB using the insert_in_db script in the package. It might be cleaner to update this package to place stored points directly into the db.

**Comment by gestom** [comment]
Waypoints without a relevant map are not that useful. If the waypoints go to the MongoDB, the maps should go as well.

**Comment by cburbridge** [comment]
Absolutely. That is why the waypoints have a map name field. This is not yet used, but see issue 11 in scitos_2d_navigation: https://github.com/strands-project/scitos_2d_navigation/issues/11.
